adoc/admin-security-role_mgmt.adoc

```diff
@@ -0,0 +1,247 @@
+
+deletecollection::
+Delete a collection of CronJob.
+
+get::
+Display individual resource.
+
+list::
+Display collections.
+
+patch::
+Update an API object in place.
+
+proxy::
+Allows running {kubectl}
+in a mode where it acts as a reverse proxy.
+
+update::
+Update fields of a resource, for example annotations or labels.
+
+watch::
+Watch resource.
+
+
+[[_sec.admin.security.role.resource]]
+== List of Resources
+
+
+This section provides an overview of the most common _resources_ used for defining roles.
+
+Autoscaler::
+https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+
+ConfigMaps::
+https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/
+
+Cronjob::
+https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/
+
+DaemonSet::
+https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/
+
+Deployment::
+https://kubernetes.io/docs/concepts/workloads/controllers/deployment/
+
+Ingress::
+https://kubernetes.io/docs/concepts/services-networking/ingress/
+
+Job::
+https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/
+
+Namespace::
+https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/
+
+Node::
+https://kubernetes.io/docs/concepts/architecture/nodes/
+
+Pod::
+https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/
+
+Persistent Volumes::
+https://kubernetes.io/docs/concepts/storage/persistent-volumes/
+
+Secrets::
+https://kubernetes.io/docs/concepts/configuration/secret/
+
+Service::
+https://kubernetes.io/docs/concepts/services-networking/service/
+
+ReplicaSets::
+https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+
+
+[[_sec.admin.security.role.create]]
+== Create Role
+
+
+Roles are defined in YAML files.
+To apply role definitions to {kube}, use `kubectl apply -f YAML_FILE`.
+The following examples provide an overview about different use cases of roles.
+
+.Simple Role for Core Resource
+====
+This example allows to `get`, `watch` and `list` all `pods` in the namespace `default`.
+
+----
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: view-pods <1>
+  namespace: default <2>
+rules:
+- apiGroups: [""] <3>
+  resources: ["pods"] <4>
+  verbs: ["get", "watch", "list"] <5>
+----
+<1>
+         Name of the role. This is required to associate the rule with
+         a group or user. For details, refer to <<_sec.admin.security.role.create_binding>>.
+
+<2>
+         Namespace the new group should be allowed to access. Use
+         `default` for {kube}' default namespace.
+
+<3>
+         {kube} API groups. Use `""` for the core group.
+
+<4>
+         {kube} resources. For a list of available resources, refer to
+         <<_sec.admin.security.role.resource>>.
+
+<5>
+         {kube} verbs. For a list of available verbs, refer to
+         <<_sec.admin.security.role.verb>>.
+
+====
+
+.Clusterwide Creation of Pods
+====
+This example allows to `create pods` clusterwide.
```

Comment:
Please review and update this section, it should be related to ClusterRole creations, why this explained about `create pods`?